File: backend/back_springboot_srvt/src/main/java/com/tecsup/back_springboot_srvt/model/Curso.java
package com.tecsup.back_springboot_srvt.model;
import java.sql.Date;

public class Curso {
    private Long id;
    private String nombre;
    private String descripcion;
    private Carrera carrera;
    private int duracion; // en horas
    private Date fecha_creacion;

}

File: frontend/front_kotlin_svrt/app/src/main/java/com/tecsup/model/Curso.kt
package com.tecsup.model

import java.time.LocalDate

data class Curso(
    val id: Long,
    val nombre: String?,
    val descripcion: String?,
    val carreraId: Long?,
    val carreraNombre: String?,
    val duracion: Int?,
    val fecha: String? // Usar String para compatibilidad con JSON
) 